### PR TITLE
Misc API changes and additions in Backend + using these

### PR DESCRIPTION
**Pull request by ostepizza**
API:
- Created a "server" route, which currently only has one endpoint: GET /serverInfo/, which returns an arbitrary value currently to show that the backend is up and running
- Created georef project endpoint for removing all points in a project, under DELETE /project/{projectId}/points.

Frontend:
- CropImage: component now takes projectId and successfully resets/removes/deletes all points in current project after crop
- Editor: Completed function to reset all markers, this function is currently only called from CropImage, and it resets all vars and arrays related to marker states
- EditorToolbar: When pressing Exit Editor and confirming, an API request is now sent to request project deletion
- UploadFile: added a loading state. The loading is displayed while attempting to contact the backend (see first point in PR), if successful the user can upload. If the backend doesn't respond properly/is down, an error message asking to try again later is displayed.

Backend, general:
- Added a clearTmpFolder-function for file helper which deletes all folders and files from the temp folder. This function is called during the API startup

### CODE REVIEW

File: backend/img2mapAPI/routers/georefProject.py [hunk 1]
```diff
@@ -84,6 +84,15 @@ async def updatePoint(projectId: int, pointId: int, point: Point):
     except Exception as e:
         raise HTTPException(status_code=404, detail=str(e))
 
+@router.delete("/{projectId}/points")
```
Comment (Trivinyx) [inline backend/img2mapAPI/routers/georefProject.py]: In restful this should be just /{projectId/point}, even tho it is multiple points becaus in restful when spesifying point it should be /.../point/{id}

File: backend/img2mapAPI/routers/server.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from fastapi import APIRouter, BackgroundTasks, HTTPException, Query #some extra imports for future use
```
Comment (Trivinyx) [inline backend/img2mapAPI/routers/server.py]: can stay but good practise is to remove unused imports for readablility and this is also increases the compiled file .pyc byte size.

File: backend/img2mapAPI/utils/core/FileHelper.py
Comment (Trivinyx) [inline backend/img2mapAPI/utils/core/FileHelper.py:1]: Nice and good addition

File: backend/img2mapAPI/utils/projectHandler.py
Comment (Trivinyx) [inline backend/img2mapAPI/utils/projectHandler.py:1]: Good

File: frontend/components/component/CropImage.tsx
Comment (Trivinyx) [inline frontend/components/component/CropImage.tsx:1]: Looks good

File: frontend/components/ui/EditorToolbar.tsx [hunk 3]
```diff
@@ -11,6 +11,7 @@ import {
 import FormModal from "@/components/ui/FormModal";
 import WarningExitModal from '@/components/ui/WarningExitModal';
 import { ViewVerticalIcon, StackIcon, CropIcon, FileTextIcon, DownloadIcon, ExitIcon } from '@radix-ui/react-icons'
+import axios from 'axios';
```
Comment (Trivinyx) [inline frontend/components/ui/EditorToolbar.tsx:14]: Why are all the axios calls not collected in one spot to be used?

**Review by Trivinyx — CHANGES_REQUESTED** [review]
Mostly good, but look at the comments for info, some need clarification and others changes

**Review by sebmid — APPROVED** [review]
Other than toms comments, v good :DDDD !!! good job

File: frontend/components/ui/EditorToolbar.tsx (hunk 3 above)
Comment (sebmid) [reply frontend/components/ui/EditorToolbar.tsx:14]: Axios calls should probably be refactored to projectApi component

**Comment by ostepizza** [comment]
I have done the suggestions except for the Axios one; there are more direct API calls in the project that have already been merged before. Instead I suggest merging now, and I will then create a separate branch to identify and refactor these (thank you for bringing these to my attention)

File: backend/img2mapAPI/routers/georefProject.py
Comment (Trivinyx) [inline backend/img2mapAPI/routers/georefProject.py:1]: Good

File: backend/img2mapAPI/routers/server.py
Comment (Trivinyx) [inline backend/img2mapAPI/routers/server.py:1]: Good

**Review by Trivinyx — APPROVED** [review]
Good adjustments and thx for adressing the Axios Issue

**Pull request merged**